### ISSUE TITLE
Elevated plasma palmitoylcarnitine, C16:0

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5976]
**Preferred term label:** Elevated plasma palmitoylcarnitine, C16:0

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Palmitoylcarnitine concentration in the blood., PMID: 8706815

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@JillMiller I am not sure we need to distinguish between C16:0 and C16:1 (#5977).
LOINC tests for simply palmitoylcarnitine
https://loinc.org/18460-6/
I think it is probably best to make terms without the C16:0/C16:1 distinctions -- with similar comments for many of the analogous terms requests. Can you provide more context if this is a mistake?

**Comment by pnrobinson** [comment]
adding term for  simply palmitoylcarnitine
